README.md

```diff
@@ -11,6 +11,9 @@ This is the Orders business logic module.
 
 ## Additional information
 
+IMPORTANT NOTE! Only in case an acquisition unit has to be assigned to the Order it is required that user should have 
+extra permission `orders-storage.acquisitions-unit-assignments.item.post` to create an purchase order.
```

Comment:
I believe this note has to be updated to mention `orders.assignments.item.post` instead of `orders-storage.acquisitions-unit-assignments.item.post`

Reply:
done